packages/camel-model/src/index.ts

```diff
@@ -1,2 +1,9 @@
-import { _apacheCamelModelVersion, _apacheCamelModel } from './camelModel'
-export { _apacheCamelModelVersion as apacheCamelModelVersion, _apacheCamelModel as apacheCamelModel }
+import definitions from './definitions-camel-model.json'
+import rests from './rests-camel-model.json'
+import dataformats from './dataformats-camel-model.json'
+import languages from './languages-camel-model.json'
+import components from './components-camel-model.json'
+
+var apacheCamelModelVersion = '3.20.2'
```

Comment:
so now is the camel model version not generated automatically anymore?  then we should set up a way to update this part from maven `version.org.apache.camel` property.

Reply:
FYI, see `replace-version` script in packages/hawtio/package.json

Reply:
This file is generated automatically from the maven generator plugin. It is still named, by default `camelModel.js`, but we rename it to `index.ts` (effectively convert it to typescript). Therefore, the camel version is retrieved from the camel-model dependency used in the model generation.